src/main/java/org/folio/repository/holdings/status/HoldingsStatusTableConstants.java

```diff
@@ -9,7 +9,7 @@
   public static final String HOLDINGS_STATUS_FIELD_LIST = String.format("%s, %s, %s", ID_COLUMN, JSONB_COLUMN, PROCESS_ID_COLUMN);
   public static final String GET_HOLDINGS_STATUS = "SELECT " + HOLDINGS_STATUS_FIELD_LIST + " from %s;";
   public static final String INSERT_LOADING_STATUS = "INSERT INTO %s (" + HOLDINGS_STATUS_FIELD_LIST + ") VALUES (%s) ON CONFLICT DO NOTHING;";
-  public static final String UPDATE_LOADING_STATUS = "UPDATE %s SET " + JSONB_COLUMN + " = '%s'::jsonb WHERE process_id=?";
```

Comment:
I've had problems with this query if json contained '?' characters, so I changed it to use '?' placeholder